### ISSUE TITLE
machines-ovirt: VCPU information is not the same in host and cluster page.

### ISSUE THREAD

**Issue by kenzhaoyihui** [cockpit-project/cockpit#9844]
Description of problem:
VCPU information is not the same in host and cluster page.
In the host page,
 
![host_vcpu](https://user-images.githubusercontent.com/18360375/44198229-6e619500-a173-11e8-83c8-c9ef3fd53ed9.png)

In cluster page,
![cluster_vcpu](https://user-images.githubusercontent.com/18360375/44198248-7e797480-a173-11e8-80a7-627eab3d9db1.png)

In xml:
![xml_vcpu](https://user-images.githubusercontent.com/18360375/44198270-89340980-a173-11e8-8cff-82693024a665.png)

Version-Release number of selected component (if applicable):
RHEL-7.6-20180814.0
cockpit-machines-ovirt-173-3.el7.noarch

How reproducible:
100%

Steps to Reproduce:
The same as the description.

Expected results:
VCPU information is  the same in host and cluster page. And it should be correct (equal to the xml information)

Additional info:

**Comment by michalskrivanek** [comment]
@bond95 can you take a look why?

**Comment by bond95** [comment]
@kenzhaoyihui information for host cockpit get from libvirt. And information for cluster - from oVirt. Thus everything works as it must be.
@michalskrivanek what do you think?

**Comment by michalskrivanek** [comment]
@bond95 I think one of them is showing wrong values;)
Probably the first one as I would presume the value "16" in sockets comes from the maximum we configure. But they are not plugged except for one of them.

**Comment by bond95** [comment]
@michalskrivanek All values are right. They are different because oVirt, for hot plugging, allocate more sockets, threads and cores. But use only some of them. That's why libvirt has topology values bigger then oVirt.

**Comment by KKoukiou** [comment]
@michalskrivanek this is long time sitting here. Is this a bug? TBH I don't understand the explanation above. Can you please elaborate?

**Comment by michalskrivanek** [comment]
@KKoukiou it shows the libvirt xml configured values, not the currently onlined cpus. This happens with ovirt all the time because we create VMs with e.g. 16 sockets. always. 
It can happen for non-ovirt VMs as well if someone plans to use hotplugging.

**Comment by KKoukiou** [comment]
@michalskrivanek thanks for the explanation.

So, to make the issue here clear, when someone hotplugs vcpus he does not see the updated UI but instead he sees the xml vcpu values. In fact I just tried it and that's partially true. I was able to see the new vcpu count just by refreshing the page after issuing the virsh setvcpu command on a running domain.
So it's just a matter about receiving some signal from libvirt for the vcpu changes.
In cockpit-machines UI though we don't yet support hotplugging/hotunplugging VCPUs so I wouldn't consider it as urgent feature.

I 'll change the tittle to match the actual generic issue for showing updated values when hotplugging VCPUs.
